src/Components/Router.jsx

```diff
@@ -0,0 +1,16 @@
+import { Routes, Route } from "react-router-dom";
+import { Home, Explore } from "../pages";
+
+const Router = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/explore" element={<Explore />} />
+      {/* <Route path="/liked" element={<Like />} />
+      <Route path="/profile" element={<Profile />} />
+      <Route path="*" element={<PageNotFound />} /> */}
```

Comment:
can remove comments if not reqd and can also ommit the use of `/` as it's not reqd in react router v6.

Reply:
Thankyou for your comments will make the required changes and commit in next pr